cli/js/web/console.ts

```diff
@@ -7,18 +7,25 @@ import { exposeForTest } from "../internals.ts";
 import { PromiseState } from "./promise.ts";
 
 type ConsoleContext = Set<unknown>;
-type InspectOptions = Partial<{
-  showHidden: boolean;
+
+/**
+ * @property `depth` Default depth of logging nested objects
+ * @property `indentLevel` Indentation level.
+ * @property `lineBreakLength` The maximum length until input values is split into multiple lines.
+ * @property `maxIterableLength` The maximum number of elements in an iterable, which would be printed.
+ * @property `strAbbreviateSize` The maximum length of the string until printed in abbreviate form.
+ * @property `minGroupLength` The minimum number of elements in an array until elements will be printed in groups.
+ */
+type InspectOptions = Partial<PrintConfig>;
```

Comment:
This is confusing, but the JS doc needs to go in `cli/js/lib.deno.ns.d.ts` rather than here. Otherwise it won't show up in `deno types` output.

Reply:
Oh, yes, that's a little bit confusing. Thank you!